### PR TITLE
Add Service Accounts to support StackDriver Logging

### PR DESCRIPTION
**Pull request by johnsonj**
Docs cleanup:
 - Add project id var for consistency (needed by terraform)
 - Fix sed commands
 - Referenced stemcell version won't deploy CF

Service Account:
 - Add a 'cf-component' service account as part of the setup
 - Apply this SA to each pool in the deployment
 - This account has the default access (editor) and the needed access for StackDriver

### CODE REVIEW

File: docs/cloudfoundry/README.md [hunk 1]
```diff
@@ -192,7 +212,7 @@ Before working this section, you must have deployed the supporting infrastructur
 1. Upload the required [Google BOSH Stemcell](http://bosh.io/docs/stemcell.html):
 
   ```
-  bosh upload stemcell https://storage.googleapis.com/bosh-cpi-artifacts/light-bosh-stemcell-3262.5-google-kvm-ubuntu-trusty-go_agent.tgz
+  bosh upload stemcell https://storage.googleapis.com/bosh-cpi-artifacts/light-bosh-stemcell-3262.12-google-kvm-ubuntu-trusty-go_agent.tgz
```
Comment (evandbrown) [inline docs/cloudfoundry/README.md]: Let's use this chance to switch to the new stemcells published on bosh.io:

` bosh upload stemcell https://bosh.io/d/stemcells/bosh-google-kvm-ubuntu-trusty-go_agent`

File: docs/cloudfoundry/cloudfoundry.yml [hunk 2]
```diff
@@ -81,6 +87,9 @@ compilation:
     root_disk_size_gb: 100
     root_disk_type: pd-ssd
     preemptible: true
+    service_account: <%= cf_service_account%>
+    service_scopes:
```
Comment (evandbrown) [inline docs/cloudfoundry/cloudfoundry.yml]: We can exclude `service_scopes`; the default will be added by the CPI.

File: docs/cloudfoundry/cloudfoundry.yml [hunk 3]
```diff
@@ -133,6 +142,9 @@ resource_pools:
       machine_type: n1-standard-4
       root_disk_size_gb: 20
       root_disk_type: pd-standard
+      service_account: <%= cf_service_account%>
+      service_scopes:
```
Comment (evandbrown) [inline docs/cloudfoundry/cloudfoundry.yml]: Remove

File: docs/cloudfoundry/cloudfoundry.yml [hunk 4]
```diff
@@ -145,6 +157,9 @@ resource_pools:
       root_disk_size_gb: 20
       root_disk_type: pd-standard
       target_pool: cf-public
+      service_account: <%= cf_service_account%>
+      service_scopes:
+        - cloud-platform
```
Comment (evandbrown) [inline docs/cloudfoundry/cloudfoundry.yml]: Remove

File: docs/cloudfoundry/cloudfoundry.yml [hunk 5]
```diff
@@ -156,6 +171,9 @@ resource_pools:
       machine_type: n1-highmem-8
       root_disk_size_gb: 100
       root_disk_type: pd-standard
+      service_account: <%= cf_service_account%>
+      service_scopes:
+        - cloud-platform
```
Comment (evandbrown) [inline docs/cloudfoundry/cloudfoundry.yml]: Remove

File: docs/cloudfoundry/cloudfoundry.yml [hunk 6]
```diff
@@ -13,6 +13,12 @@ google_zone = "{{ZONE}}"
 network = "cf"
 public_subnetwork = "cf-public-#{google_region}"
 private_subnetwork = "cf-private-#{google_region}"
+
+# Google service account settings
+project_id = "{{PROJECT_ID}}"
+org, project = project_id.split(":")
+project_domain_prefix = [project, org].compact.join(".")
+cf_service_account = "cf-component@#{project_domain_prefix}.iam.gserviceaccount.com"
```
Comment (evandbrown) [inline docs/cloudfoundry/cloudfoundry.yml]: Nice! How'd you figure out the `project.org` format?

Comment (johnsonj) [inline docs/cloudfoundry/cloudfoundry.yml]: just scratching my head at the console output, i'm pretty certain it's accurate though. this somewhat obfuscated code let's us handle both org/non org projects

**Comment by johnsonj** [comment]
@evandbrown ready for review

**Pull request merged**